site/generate.mjs

```diff
@@ -311,14 +301,20 @@ walkStruct(struct);
     //
     "host-gc-required": 99,
 
+    //16
+    "set-methods": 99,
```

Comment:
this doesn't look right?

Reply:
`set-methods` is merged into the spec but `99` is the marker for showing as ESNext on the site, which i kept for this feature since 16 isnt out yet

Reply:
Ah, let's clarify the comment a bit then :)